### DISCUSSION TITLE
Welcome to Xamarin.Forms.InputKit Discussions!

### DISCUSSION THREAD

**Discussion by enisn** [enisn/Xamarin.Forms.InputKit#223] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anshumaansb** [comment]
Any alternate for dropdown in maui

**Reply by enisn** [reply]
Working in UraniumUI, but still not completed

https://github.com/enisn/UraniumUI/pull/670
